pages/forgotPassword.js

```diff
@@ -3,8 +3,58 @@ import { Segment, Form, Header, Button } from 'semantic-ui-react';
 import cx from 'classnames';
 import forgotPasswordStyles from '@app/styles/forgotPassword.module.css';
 import LabeledInput from '@app/components/pieces/labeledInput';
+import { useEffect, useState } from 'react';
+import { usePost } from 'swirl-react';
+import _ from 'lodash';
+import validator from 'validator';
+
+export default function ForgotPassword() {
+  const [email, setEmail] = useState('');
+
+  //Form validation
+  const [showEmailError, setShowEmailError] = useState(false);
+
+  const { isLoading, data, error, trigger } = usePost({
+    url: 'api.connectorlando.tech/auth/forgot-password',
+    body: {
+      email: email,
+    },
+  });
+
+  function handleEmailInput(event) {
+    setEmail(event?.target?.value?.trim());
+  }
+
+  //log data and error
+  useEffect(() => {
+    if (data) {
+      console.log(data);
+    }
+
+    if (error) {
+      console.log(error);
+    }
+  }, [data, error]);
+
+  //Check if submit button should be enabled
+  const isButtonEnabled = !_.isEmpty(email) && isEmailValid();
+
+  //Validate email everytime it changes
+  useEffect(() => {
+    if (isEmailValid()) {
+      setShowEmailError(false);
+    }
+  });
```

Comment:
`useEffect` is for running a code block whenever a variable is updated. In this case, you want to show the email error if the email is not valid. So you have to add the `email` as a watched property
```suggestion
  //Validate email everytime it changes
  useEffect(() => {
    if (isEmailValid()) {
      setShowEmailError(false);
    }
  }, [email]);
```